### PR TITLE
Update CHANGELOG and version mapping for v1.4.3

### PR DESCRIPTION
**Pull request by rramkumar1**
/assign @MrHohn

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+
+**Merged pull requests:**
+
+- reflect readiness probe in health check for NEG enabled ClusterIP service backend \[release-1.4\] [\#633](https://github.com/kubernetes/ingress-gce/pull/633) ([freehan](https://github.com/freehan))
+- do not leak LB when ingress class is changed \[release-1.4\] [\#626](https://github.com/kubernetes/ingress-gce/pull/626) ([agau4779](https://github.com/agau4779))
+- Configure leader election with completely separate k8s client \[release-1.4\] [\#624](https://github.com/kubernetes/ingress-gce/pull/624) ([rramkumar1](https://github.com/rramkumar1))
+
+# Change Log
+
+## [v1.4.3](https://github.com/kubernetes/ingress-gce/tree/v1.4.3) (2019-02-12)
```
Comment (MrHohn) [inline CHANGELOG.md]: Seems like we have two v1.4.3 sections?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+
+**Merged pull requests:**
+
+- reflect readiness probe in health check for NEG enabled ClusterIP service backend \[release-1.4\] [\#633](https://github.com/kubernetes/ingress-gce/pull/633) ([freehan](https://github.com/freehan))
+- Cherry-picking \#547 onto 1.4 branch [\#626](https://github.com/kubernetes/ingress-gce/pull/626) ([agau4779](https://github.com/agau4779))
+- Configure leader election with completely separate k8s client \[release-1.4\] [\#624](https://github.com/kubernetes/ingress-gce/pull/624) ([rramkumar1](https://github.com/rramkumar1))
+
+# Change Log
+
+## [v1.4.3](https://github.com/kubernetes/ingress-gce/tree/v1.4.3) (2019-02-12)
```
Comment (MrHohn) [inline CHANGELOG.md]: Humm..Actually three

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,67 @@
...
+[Full Changelog](https://github.com/kubernetes/ingress-gce/compare/v1.4.2...v1.4.3)
+
+**Implemented enhancements:**
+
+- BackendConfig security policy not enforced  [\#616](https://github.com/kubernetes/ingress-gce/issues/616)
+- original http request origin and host headers are overriden [\#179](https://github.com/kubernetes/ingress-gce/issues/179)
```
Comment (MrHohn) [inline CHANGELOG.md]: These two might not be implemented enhancements?

File: CHANGELOG.md (hunk 1 above)
Comment (rramkumar1) [reply CHANGELOG.md]: Oops I ran the tool multiple times and it seems to have appended. Fixed.

**Pull request merged**